src/modules/xcm/index.ts

```diff
@@ -112,7 +112,7 @@ export const xcmChainObj: XcmChainObj = {
   [Chain.KARURA]: {
     name: Chain.KARURA,
     relayChain: Chain.KUSAMA,
-    img: 'https://polkadot.js.org/apps/static/karura.6540c949..svg',
+    img: 'https://assets.coingecko.com/coins/images/17172/small/karura.jpeg?1626782066',
```

Comment:
What about having all logos in the astar-apps or astar-ui if you think this is more appropriate? If coin gecko decide to change logos location this will break our display

Reply:
@bobo-k2 Fixed int [this](https://github.com/AstarNetwork/astar-apps/pull/688/commits/26861968521f3800a87d366fdb7d2b2e5f762342) commit.